codegen/src/main.rs

```diff
@@ -106,7 +107,7 @@ fn load_file<P: AsRef<Path>>(name: P, features: &Tokens, lookup: &mut Lookup) ->
                 // We don't want to try to load the generated rust files and
                 // parse them, so we ignore them here.
                 for name in IGNORED_MODS {
-                    if item.ident == name {
+                    if item.ident.to_string() == *name {
```

Comment:
I opened https://github.com/alexcrichton/proc-macro2/pull/92 to avoid these changes.